app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -6,12 +6,13 @@ class Api::V2::ContentViewFilterRulesController < Api::V2::ApiController
     api :GET, "/content_view_filters/:content_view_filter_id/rules", N_("List filter rules")
     param :content_view_filter_id, :identifier, :desc => N_("filter identifier"), :required => true
     def index
-      options = sort_params
-      options[:load_records?] = true
-      options[:filters] = [{ :terms => { :id => ContentViewFilter.rule_ids_for(@filter) } }]
-
-      @search_service.model = ContentViewFilter.rule_class_for(@filter)
-      respond(:collection => item_search(ContentViewFilter.rule_class_for(@filter), params, options))
+      results = ContentViewFilter.rule_class_for(@filter)
```

Comment:
It doesn't look like you're limiting filter rules to a specific filter?  this would return all rules across the whole server wouldn't it?